report/templates/report.html

```diff
@@ -53,43 +53,146 @@ <h2 class="header-titles__url">{{ url }}</h2>
             In this report:
           </div>
           <ul class="menu__nav">
-            {{#each aggregations}}
             <li class="menu__nav-item">
-              <a class="menu__link" href="#{{ generateAnchor this.shortName }}">
-                <span class="menu__link-label">{{ this.shortName }}</span>
-                <span class="menu__link-score menu__link-score--{{ getItemRating this.score.overall }}">{{ convertToPercentage this.score.overall }}</span>
+              <a class="menu__link" href="#progressive-web-app">
+                Progressive Web App
+              </a>
+            </li>
+
+            <li class="menu__nav-item">
+              <a class="menu__link" href="#best-practices">
+                Best Practices
               </a>
             </li>
-            {{/each}}
           </ul>
         </div>
       </div>
       <div class="js-breakdown report-body__breakdown">
-        {{#each aggregations}}
-          <section class="report-body__breakdown-item" id="{{ generateAnchor this.shortName }}">
-            <h1 class="report-section__title">
-              <span class="report-section__label">{{ this.name }}</span>
-              <span class="report-section__score report-section__score--{{ getItemRating this.score.overall }}">{{ convertToPercentage this.score.overall }}</span>
-            </h1>
-
-            <h2 class="report-section__subtitle">Individual results:</h2>
+        <h1 class="report-section__title" id="progressive-web-app">Progressive Web App</h1>
+        <p class="report-section__description">
+          These audits check for various aspects of creating a Progressive Web App, and they affect your overall score.
+        </p>
+
+        <div class="toggle">
+          <h1 class="toggle__title">Toggle view:</h1>
+
+          <label class="toggle__label" for="user-feature">User feature</label>
+          <input name='view-toggle' class="js-toggle-user" id="user-feature" type='radio' value='user' checked>
```

Comment:
I think his point was you use double and single quotes here, and to switch to all double